### ISSUE TITLE
build fails.  errors in sled 0.16.11.

### ISSUE THREAD

**Issue by dan-da** [hermits-grove/hermitdb#27]
hi, Is there some trick to make this build properly?   I didn't see any build/install instructions, so I just cloned this repo and ran cargo build.  sled version = 0.16.11 is specified in the Cargo.toml, so I guess it shouldn't have changed...?   Anyway, I get these errors:

```
   Compiling sled v0.16.11
error[E0599]: no method named `clone` found for type `crossbeam_epoch::guard::Guard` in the current scope
   --> /home/user/.cargo/registry/src/github.com-1ecc6299db9ec823/sled-0.16.11/src/iter.rs:110:40
    |
110 |             let pin_guard = self.guard.clone();
    |                                        ^^^^^ method not found in `crossbeam_epoch::guard::Guard`

error[E0599]: no method named `clone` found for type `crossbeam_epoch::guard::Guard` in the current scope
   --> /home/user/.cargo/registry/src/github.com-1ecc6299db9ec823/sled-0.16.11/src/iter.rs:111:40
    |
111 |             let get_guard = self.guard.clone();
    |                                        ^^^^^ method not found in `crossbeam_epoch::guard::Guard`

error[E0599]: no method named `clone` found for type `crossbeam_epoch::guard::Guard` in the current scope
   --> /home/user/.cargo/registry/src/github.com-1ecc6299db9ec823/sled-0.16.11/src/iter.rs:262:40
    |
262 |             let pin_guard = self.guard.clone();
    |                                        ^^^^^ method not found in `crossbeam_epoch::guard::Guard`

error[E0599]: no method named `clone` found for type `crossbeam_epoch::guard::Guard` in the current scope
   --> /home/user/.cargo/registry/src/github.com-1ecc6299db9ec823/sled-0.16.11/src/iter.rs:263:40
    |
263 |             let get_guard = self.guard.clone();
    |                                        ^^^^^ method not found in `crossbeam_epoch::guard::Guard`

error[E0599]: no method named `clone` found for type `crossbeam_epoch::guard::Guard` in the current scope
  --> /home/user/.cargo/registry/src/github.com-1ecc6299db9ec823/sled-0.16.11/src/pinned_value.rs:16:44
   |
16 |         PinnedValue(self.0, self.1, self.2.clone())
   |                                            ^^^^^ method not found in `crossbeam_epoch::guard::Guard`

error[E0599]: no method named `clone` found for type `crossbeam_epoch::guard::Guard` in the current scope
   --> /home/user/.cargo/registry/src/github.com-1ecc6299db9ec823/sled-0.16.11/src/tree.rs:379:34
    |
379 |         let double_guard = guard.clone();
    |                                  ^^^^^ method not found in `crossbeam_epoch::guard::Guard`

error[E0599]: no method named `clone` found for type `crossbeam_epoch::guard::Guard` in the current scope
   --> /home/user/.cargo/registry/src/github.com-1ecc6299db9ec823/sled-0.16.11/src/tree.rs:418:34
    |
418 |         let double_guard = guard.clone();
    |                                  ^^^^^ method not found in `crossbeam_epoch::guard::Guard`

error[E0599]: no method named `clone` found for type `crossbeam_epoch::guard::Guard` in the current scope
   --> /home/user/.cargo/registry/src/github.com-1ecc6299db9ec823/sled-0.16.11/src/tree.rs:488:34
    |
488 |         let double_guard = guard.clone();
    |                                  ^^^^^ method not found in `crossbeam_epoch::guard::Guard`

error[E0599]: no method named `clone` found for type `crossbeam_epoch::guard::Guard` in the current scope
   --> /home/user/.cargo/registry/src/github.com-1ecc6299db9ec823/sled-0.16.11/src/tree.rs:575:35
    |
575 |             let pin_guard = guard.clone();
    |                                   ^^^^^ method not found in `crossbeam_epoch::guard::Guard`

error[E0599]: no method named `clone` found for type `crossbeam_epoch::guard::Guard` in the current scope
   --> /home/user/.cargo/registry/src/github.com-1ecc6299db9ec823/sled-0.16.11/src/tree.rs:659:38
    |
659 |             let double_guard = guard.clone();
    |                                      ^^^^^ method not found in `crossbeam_epoch::guard::Guard`

error[E0599]: no method named `clone` found for type `crossbeam_epoch::guard::Guard` in the current scope
   --> /home/user/.cargo/registry/src/github.com-1ecc6299db9ec823/sled-0.16.11/src/tree.rs:749:34
    |
749 |         let double_guard = guard.clone();
    |                                  ^^^^^ method not found in `crossbeam_epoch::guard::Guard`

error: aborting due to 11 previous errors
```

```
$ rustc --version
rustc 1.40.0 (73528e339 2019-12-16)
```

**Comment by davidrusu** [comment]
Unfortunately, I ran out of time and had to drop this project. I have to do a deep dive to refresh my memory on what state I left things in, but I remember the core ideas were implemented. I was in the process of building out a password manager on top of HermitDB and using that usecase to refactor and improve the development experience of working with this database.

I'll see about getting things building once again :)
